js/util.js

```diff
@@ -254,28 +249,35 @@
             this.funcToCall.apply(ctx, this.inputPoints);
         }
         else{
-            console.log(this.funcToCall);
             this.funcToCall.apply(ctx, new Array());
         }
 
     }
 
 
-    //Pathset
-    function Pathset(pathArray){
+    //Shape
+    function Shape(pathArray){
         var len = pathArray.length;
         var i = 0;
         while (i<len){
             if(!(pathArray[i] instanceof Path)){
-                throw new Error('Only paths may be added to a Pathset, ' + pathArray[i] + " is not.");
+                throw new Error('Only paths may be added to a Shape, ' + pathArray[i] + " is not.");
             }
+            i = i+1;
         }
 
         this.pathArray = pathArray;
     }
-    Pathset.prototype.getPathArray = function(){
+    Shape.prototype.getPathArray = function(){
```

Comment:
FYI, Java-style getters aren't really a thing in JavaScript land, so this one is pretty much superfluous. Plus, `pathArray` doesn't actually reference to the `Shape` instance's property called `pathArray` (`this.pathArray` is what would do it).